server/database/models/ModelDocumentRepository.js

```diff
@@ -0,0 +1,78 @@
+const AbstractRepository = require("./AbstractRepository");
+
+class ModelDocumentRepository extends AbstractRepository {
+  constructor() {
+    // Call the constructor of the parent class (AbstractRepository)
+    // and pass the table name "Model_Docs" as configuration
+    super({ table: "Model_Docs" });
+  }
+
+  // The C of CRUD - Create operation
+  async create(model) {
+    // Execute the SQL INSERT query to add a new model to the "Model_docs" table
+    const [result] = await this.database.query(
+      `insert into ${this.table} (Type_Doc,Languages_source,Status,Real_Path_Emplacement,Id_Client,Id_Translator) values (?, ?, ?, ?, ?, ?)`,
```

Comment:
table names are in snake case, some fields are not foreseen in this table, others are relevant but are not listed in your MPD